app/src/ui/preferences/prompts.tsx

```diff
@@ -226,6 +234,18 @@ export class Prompts extends React.Component<
             onSelected={this.onUncommittedChangesStrategyChanged}
           />
         </div>
+        <div className="advanced-section">
+          <h2>Style Guides</h2>
```

Comment:
```suggestion
          <h2>Commit Length</h2>
```

Until we have other settings that could be logically grouped under a shared heading, lets go ahead and be specific to the setting here for easy scanning. This particular setting was already hard to place under the current settings panels.. not really "advanced" and not really "prompts", but prompts are warnings as is this.